### ISSUE TITLE
Não é possível cadastrar um usuario pelo BIGSEA

### ISSUE THREAD

**Issue by fonluiz** [analytics-ufcg/bigsea-melhorbusao#7]
Quando tenta-se cadastrar um novo usuário pelo BigSea, o aplicativo diz que o usuário já existe, mesmo que não exista.

**Comment by paulo308** [comment]
Esta situação é referente ao serviço de autenticação do BigSea (eubrabigsea.dei.uc.pt)?

**Comment by fonluiz** [comment]
Não. É referente à aplicação "Melhor Busão". O cadastro com o BigSea parece estar funcionando normalmente.
